### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by JoshMacSween**

### CODE REVIEW

**Comment by JoshMacSween** [comment]
Still super bare bones, let me know what you think :)

File: README.md [hunk 1]
```diff
@@ -2,3 +2,32 @@
...
+  
+  Fire up new PostgreSQL server 
+  
+  <code>pg_ctl -D /usr/local/var/postgres start</code>
+  
+  Create Database 
+  
+  <code>initdb /usr/local/var/postgres</code>
+  
+  And that should do it. Any issues can be forwarded to alan@alanvardy.com
```
Comment (alanvardy) [inline README.md]: I am totally getting thrown under the bus here :laughing:

Comment (JoshMacSween) [reply README.md]: Yep :)

**Comment by alanvardy** [comment]
Can you add what's missing to the issue? Things like Linux install instructions, how to open a pull request etc.

**Pull request merged**